contribute.json

```diff
@@ -0,0 +1,34 @@
+{
+    "name": "MIG: Mozilla InvestiGator",
+    "description": "Mozilla's real-time digital forensics and investigation platform.",
+    "repository": {
+        "url": "https://github.com/mozilla/mig",
+        "license": "MPL2",
+        "tests": "https://travis-ci.org/mozilla/mig"
+    },
+    "participate": {
+        "home": "https://mig.mozilla.org/",
+        "docs": "https://mig.mozilla.org/doc.html",
+        "irc": "irc://irc.mozilla.org/#infosec",
```

Comment:
MIG has an IRC channel as well (#mig) but this is probably OK too

Reply:
I'm fine with fixing, didn't know.

Reply:
Fixed in cb77c6121b373a492fc59f81132fd6f0f344979d